### DISCUSSION TITLE
AllowedInclude::callback does not exist

### DISCUSSION THREAD

**Discussion by BenjaminPichon** [spatie/laravel-query-builder#905] [Q&A]
Hello,
I'm using the v5 of spatie and I'm trying to use an allowedInclude::callback, but it seems that it doesn't exist on the allowedInclude file,
does anyone had this probleme and know how to fix this ?
Thanks in advance!

**Comment by ajagabos007** [comment]
I am having similar issues this day. Any possibility you found a solution to it?
